packages/repeater/src/lib/RepeaterFactory.ts

```diff
@@ -38,4 +37,18 @@ export class RepeaterFactory {
       configuration: this.configuration
     });
   }
+
+  private registerRunnerOptions(
+    options: RequestRunnerOptions | undefined
+  ): DependencyContainer {
+    const container = this.configuration.container.createChildContainer();
+
+    if (options) {
+      container.register(RequestRunnerOptions, { useValue: options });
+    }
```

Comment:
seems it's better to move definition/registration of default options from `request-runner/register` here to handle them in one place. also imo better to treat passed options as partial over defaults.